### PR TITLE
Add a util function to serialize an arbitrary model

### PR DESCRIPTION
**Pull request by tnsengimana**

### CODE REVIEW

**Comment by hamedahmadi** [comment]
@rfadams @tnsengimana I don't think this is a good approach. We are dumping all of are data blindly into the index. This not only is exposing too much and out of our control, it provides very little benefit. All these fields will not be searchable, and will only cause the index to grow unnecessary big. We should not be treating the index as another copy of our database. It should just be a way to find records faster. When we know what record we need, they should be fetched using proper channels. This approach is creating a loop hole for all of our permissions.

**Comment by tnsengimana** [comment]
@hamedahmadi I think you are mixing two different concepts here. CUD events are independent of indexing. That is, an emitted CUD event is unaware of services that will consume it. It is the responsibility of the consuming service to decide what to do with data included in the event.

If I was to rephrase your concern, I would say: is it a good approach to include all of the model's attributes when a CUD event is emitted? I'd say yes since we don't know the scope of attributes needed by the consuming service. 

`mp-indexer` on the other hand, relays on CUD events to make its task of creating & updating Algolia index much easier. If you look at the implementation of `mp-indexer`, you will notice that most attributes included in a CUD event are not necessarily sent to Algolia. 

cc @rfadams

**Comment by rfadams** [comment]
I think we have to separate the needs of `mp-indexer` from how we are going to do events in general. Let's discuss on Friday morning.

File: zc_common/tests.py [hunk 1]
```diff
@@ -0,0 +1,11 @@
+from django.test.runner import DiscoverRunner
```
Comment (hamedahmadi) [inline zc_common/tests.py]: @tnsengimana We already have this  https://github.com/ZeroCater/zc_common/blob/master/zc_common/databaseless_test_runner.py

Comment (tnsengimana) [inline zc_common/tests.py]: Oh couldn't have noticed. Thanks @hamedahmadi

**Comment by tnsengimana** [comment]
[Clean up legacy, mp-indexer to make them prod ready](https://trello.com/c/il0qW5bT/1727-clean-up-legacy-mp-indexer-to-make-them-prod-ready)

**Pull request merged**